src/bci_build/package/apache_tomcat.py

```diff
@@ -0,0 +1,82 @@
+"""Provide an Apache Tomcat container."""
+
+from bci_build.package import CAN_BE_LATEST_OS_VERSION
+from bci_build.package import DOCKERFILE_RUN
+from bci_build.package import _SUPPORTED_UNTIL_SLE
+from bci_build.package import OsVersion
+from bci_build.package import ParseVersion
+from bci_build.package import Replacement
+
+from .appcollection import ApplicationCollectionContainer
+
+_TOMCAT_VERSIONS: list[int] = [9, 10]
+assert _TOMCAT_VERSIONS == sorted(_TOMCAT_VERSIONS)
+
+TOMCAT_CONTAINERS = [
+    ApplicationCollectionContainer(
+        name="apache-tomcat",
+        pretty_name=f"Apache Tomcat {tomcat_major}",
+        package_name=f"apache-tomcat-{tomcat_major}-image"
+        if os_version.is_tumbleweed
+        else f"sac-apache-tomcat-{tomcat_major}-image",
+        os_version=os_version,
+        is_latest=(
+            (os_version in CAN_BE_LATEST_OS_VERSION)
+            and tomcat_major == _TOMCAT_VERSIONS[-1]
+            and os_version.is_tumbleweed
```

Comment:
This condition drops the latest tag from SP6, is that intentional?

Reply:
yes, we talked about that already, appcol doesn't have `:latest tags

Reply:
:scream: